### PR TITLE
THORN-2169: Unify useUberJar maven plugin property name.

### PR DESCRIPTION
**Pull request by mkouba**
- "wildfly-swarm.useUberJar" -> "swarm.useUberJar"
- old property name is still supported
- simplify the usage: e.g. "-Dswarm.useUberJar" instead of
"-Dwildfly-swarm.useUberJar=true"

### CODE REVIEW

**Comment by mkouba** [comment]
@tradej Feel free to improve the docs part ;-)

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc [hunk 1]
```diff
@@ -262,18 +262,20 @@ A file path where to store the `stdout` output instead of sending it to the `std
...
 
 |Default
-|false
+|
 
 |Used by
 |`run`, `start`
 |===
++
+NOTE: Up to version 2.2.0.Final the property name was `wildfly-swarm.useUberJar` and only the value `true` enabled the property: `-Dwildfly-swarm.useUberJar=true`. This is no longer the case and any value enables this property, including an empty string: `-Dswarm.useUberJar`. The old property name still works though.
```
Comment (Ladicek) [inline docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: Based on my reading of the code, this is not entirely true. With the new name, you can pass anything, but if the old name is used, it must still be passed `true`.

Comment (Ladicek) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: I'd rephrase to:

NOTE: Up to version 2.2.0.Final, this property was called `wildfly-swarm.useUberJar` and only the value `true` enabled this behavior: `-Dwildfly-swarm.useUberJar=true`. You can continue to use the old name without change, but we encourage you to migrate to the new variant which doesn't require setting any value: `-Dswarm.useUberJar`.

Comment (mkouba) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: > but if the old name is used, it must still be passed true

Well, that's what I've tried to express ;-). Your wording reads much better. Pls add a commit - PR is open.

Comment (Ladicek) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: Done, thanks.

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc [hunk 2]
```diff
@@ -262,18 +262,19 @@ A file path where to store the `stdout` output instead of sending it to the `std
...
 |Default
-|false
+|
 
 |Used by
 |`run`, `start`
 |===
-
++
+NOTE: Up to version 2.2.0.Final, the property was called `wildfly-swarm.useUberJar` and only the value `true` enabled this behavior: `-Dwildfly-swarm.useUberJar=true`. You can continue to use the old name without change, but we encourage you to migrate to the new variant which doesn't require setting any value: `-Dswarm.useUberJar`.
```
Comment (tradej) [inline docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: Please perform the following changes:

* Use one sentence per line. (It's a convention, plus it makes content easier to manipulate.)
* Change the first sentence to:
   _Before version 2.2.0.Final, this property was called `wildfly-swarm.useUberJar`, and only setting it to `true` enabled this behavior: `-Dwildfly-swarm.useUberJar=true`._
* Change the second sentence to:
   _You can continue using the old name, but consider using the new variant, which does not require setting a value: `-Dswarm.useUberJar`._

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc [hunk 3]
```diff
@@ -262,18 +262,19 @@ A file path where to store the `stdout` output instead of sending it to the `std
 |===
 
 useUberJar::
-If true, the `-thorntail.jar` file specified at `${project.build.directory}` is used.
+If specified, the `-thorntail.jar` file located in `${project.build.directory}` is used.
```
Comment (tradej) [inline docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc:265]: Please change to _...located in the..._

**Review by tradej — CHANGES_REQUESTED** [review]
Small changes to docs requested inline.

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc (hunk 2 above)
Comment (mkouba) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: Feel free to add a commit with updates ;-)

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc (hunk 3 above)
Comment (Ladicek) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc:265]: I think that would read worse. IMHO, it's either "located in ${project.build.directory}" or "located in the ${project.build.directory} directory", because ${project.build.directory} denotes a filesystem path. I find the 2nd option worse.

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc (hunk 2 above)
Comment (Ladicek) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc]: I did that.

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc (hunk 3 above)
Comment (tradej) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc:265]: In CCS, there is an informal convention that if a variable name already contains the word _directory,_ it's not necessary to repeat it, but I guess it can stay as it is as well.

**Review by tradej — APPROVED** [review]

File: docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc (hunk 3 above)
Comment (Ladicek) [reply docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc:265]: That's exactly why I wouldn't use my 2nd alternative from my previous post. At the same time, I find adding `the` because ${project.build.directory} is a variable expansion, not a normal part of the sentence. I guess in cases like this, personal tase matters a lot :-)

**Comment by mkouba** [comment]
retest this please

**Comment by Ladicek** [comment]
Windows test suite failures are unrelated (MP Config and MP Metrics TCKs).
